next/backend/utils/forms.ts

```diff
@@ -238,3 +239,42 @@ export const getEform = (id: string | string[] | undefined): EFormValue => {
   if (!eform) throw new Error(`Invalid form name - validateFormName returned: ${formSlug}`)
   return eform
 }
+
+// mock-up data
```

Comment:
would move these functions to api.ts - also consider placing the mock data itself in a separate file (mocks.ts ?)

also, might want to sync with @matejpeluha who's reorganising these files as we speak